packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -475,19 +530,106 @@ class _ModalBottomSheetRoute<T> extends PopupRoute<T> {
        assert(isDismissible != null),
        assert(enableDrag != null);
 
+  /// A builder for the contents of the sheet.
+  ///
+  /// The bottom sheet will wrap the widget produced by this builder in a
+  /// [Material] widget.
   final WidgetBuilder? builder;
```

Comment:
This should not be null, it seems like a mistake we made when we migrate to null safety. and it should probably be a required parameter in constructor

Reply:
I already wondered why this is nullable, thanks for letting me know.